callisto/delivery/views.py

```diff
@@ -224,6 +226,7 @@ def export_as_pdf(request, report_id, force_download=True, filename='report.pdf'
             except Exception:
                 logger.exception("could not export report {}".format(report_id))
                 form.add_error(None, "There was an error exporting your report.")
+                context.update({'generate_pdf_error': True})
```

Comment:
for these, you should be able to get the form from the context and verify that the correct error was added to it in line 228